added topics on Error and Error Handling

araut1 opened the pull request:
Issue #2

araut1 committed:
#2 : added topics on Error and Error Handling

Added information about types of error and how to deal with them. Language and example are provided with beginer developers in mind.

araut1 committed:
Update README.md

annu12340 commented on docs/README.md:
```diff
@@ -14,7 +14,17 @@
+- Error Handling
+=======
 - Difference between innerHTML and innerText
+>>>>>>> d9076e12a32667ed37ebb08e8550853a6c7926be
```
Remove this line

annu12340 commented on docs/README.md:
```diff
@@ -310,13 +320,74 @@ https://codeburst.io/javascript-passing-by-value-vs-reference-explained-in-plain
+- calling a non-existing method
+- wrong source code file name
+
+In the case of JavaScript, compile-time errors occur at interpret time. For example, in the below code, a syntax error occurs because it is missing semi-colon ';' at the end of the print() function invocation.
```
In js ; are optional. So, I think this couldn't result in an error
Try adding another example

annu12340 commented on docs/README.md:
```diff
@@ -310,13 +320,74 @@ https://codeburst.io/javascript-passing-by-value-vs-reference-explained-in-plain
 
+## 7. Errors
+
+![error](https://i.ibb.co/jD2kG28/e.jpg)
```
Nice memes 😆

annu12340 commented on docs/README.md:
```diff
@@ -310,13 +320,74 @@ https://codeburst.io/javascript-passing-by-value-vs-reference-explained-in-plain
+
+![error](https://i.ibb.co/jD2kG28/e.jpg)
+
+""To err is human..."  Everyone makes a mistake, even the seasoned developer. Hence, it's essential to know about different errors and how to solve them. Errors can be divided into two groups: compile-time errors and run-time errors.
```
The rest of the work looks really good👏 👏 👏

araut1 committed:
Fixed compile-time error syntax example + removed dummy value

A. Removed dummy value in the index
    section.

B. Provided suitable example for  compile-time error
